common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -2469,6 +2475,23 @@ object CelebornConf extends Logging {
       .timeConf(TimeUnit.MILLISECONDS)
       .createWithDefaultString("10min")
 
+  val WORKER_DECOMMISSION_CHECK_INTERVAL: ConfigEntry[Long] =
+    buildConf("celeborn.worker.decommission.check.interval")
+      .categories("worker")
+      .doc(
+        "The wait interval of checking whether all the shuffle expired during worker decomission")
+      .version("0.2.0")
+      .timeConf(TimeUnit.MILLISECONDS)
+      .createWithDefaultString("30s")
+
+  val WORKER_DECOMMISSION_CHECK_TIMEOUT: ConfigEntry[Long] =
+    buildConf("celeborn.worker.decommission.check.timeout")
+      .categories("worker")
+      .doc("The wait time of waiting for all the shuffle expire during worker decommission.")
+      .version("0.2.0")
+      .timeConf(TimeUnit.MILLISECONDS)
+      .createWithDefaultString("3d")
```

Comment:
Maybe too long? 2h should be enough for small/middle scale

Reply:
In our prod, it's always 2d...